api/infrastructure/deploy-api.sh

```diff
@@ -90,53 +66,29 @@ case $key in
 esac
 done
 
-if [ -z "${S3_BUCKET}" ] || [ -z "${ECR_REPO}" ] || [ -z "${AWS_DEFAULT_REGION}" ] ; then
+if [ -z "${S3_BUCKET}" ] || [ -z "${AWS_DEFAULT_REGION}" ] ; then
     echo "$usage" >&2
     exit 1
 fi
 
 ACCOUNT_ID=$(aws sts get-caller-identity --query "Account" --output text)
-ECR_ENDPOINT="${ACCOUNT_ID}.dkr.ecr.${AWS_DEFAULT_REGION}.amazonaws.com"
 S3_UPLOAD_URI="s3://${S3_BUCKET}/api/ParallelCluster.openapi.yaml"
-
-echo "Building docker image"
-"${SCRIPT_DIR}/../docker/awslambda/docker-build.sh"
-
-echo "Pushing docker image to ${ECR_ENDPOINT}/${ECR_REPO}"
-aws ecr get-login-password | docker login --username AWS --password-stdin "${ECR_ENDPOINT}"
-docker tag pcluster-lambda:latest "${ECR_ENDPOINT}/${ECR_REPO}:latest"
-docker push "${ECR_ENDPOINT}/${ECR_REPO}:latest"
+POLICIES_S3_URI="s3://${S3_BUCKET}/stacks/parallelcluster-policies.yaml"
+POLICIES_TEMPLATE_URI="http://${S3_BUCKET}.s3.${AWS_DEFAULT_REGION}.amazonaws.com/stacks/parallelcluster-policies.yaml"
 
 echo "Publishing OpenAPI specs to S3"
 aws s3 cp "${SCRIPT_DIR}/../spec/openapi/ParallelCluster.openapi.yaml" "${S3_UPLOAD_URI}"
 
-echo ""
-if [ "$SKIP_IMAGE_IMPORT" = "true" ]; then
-  echo "Uploaded image will be used directly"
-  OVERRIDE_IMAGE_URI_PRM="EcrImageUri"
-  ECR_IMAGE_URI="${ECR_ENDPOINT}/${ECR_REPO}:latest"
-else
-  echo "Uploaded image will simulate a public image and will be therefore imported into a new private image"
-  OVERRIDE_IMAGE_URI_PRM="PublicEcrImageUri"
-fi
+echo "Publishing policies CloudFormation stack to S3"
+aws s3 cp "${SCRIPT_DIR}/../../cloudformation/policies/parallelcluster-policies.yaml" "${POLICIES_S3_URI}"
 
 echo "Deploying API template"
 aws cloudformation deploy \
     --stack-name "${STACK_NAME}" \
     --template-file "${SCRIPT_DIR}/parallelcluster-api.yaml" \
     --s3-bucket "${S3_BUCKET}" \
     --s3-prefix "api/" \
-    --parameter-overrides ApiDefinitionS3Uri="${S3_UPLOAD_URI}" "${OVERRIDE_IMAGE_URI_PRM}"="${ECR_ENDPOINT}/${ECR_REPO}:latest" \
+    --parameter-overrides ApiDefinitionS3Uri="${S3_UPLOAD_URI}" \
+                          PoliciesTemplateUri="${POLICIES_TEMPLATE_URI}" \
                           EnableIamAdminAccess="${ENABLE_IAM_ADMIN}" CreateApiUserRole="${CREATE_API_USER}" \
-                          ImageBuilderVpcId="${IMAGE_BUILDER_VPC_ID}" \
-                          ImageBuilderSubnetId="${IMAGE_BUILDER_SUBNET_ID}" \
     --capabilities CAPABILITY_NAMED_IAM
-
```

Comment:
question, non-blocking, should we add the layer to the lambda too?

Reply:
I just got confused `parallelcluster-api.yaml` itself adds the layer to the lambda

Anyway there is still a problem. From this PR I see `parallelcluster-api.yaml` as deleted

Reply:
My fault here, was confused by GH